rebasehelper/output_tools/text_output_tool.py

```diff
@@ -172,6 +175,14 @@ def print_checkers_text_output(cls, checkers_results):
                 if check == check_tool.get_checker_name():
                     logger_report.info('\n'.join(check_tool.format(data)))
 
+    @classmethod
+    def print_build_log_hooks_result(cls, build_log_hooks_result):
+        for build_log_hook in six.itervalues(build_log_hook_runner.build_log_hooks):
+            for hook, data in six.iteritems(build_log_hooks_result):
```

Comment:
I don't understand why this loop is here. Can't you simply use `build_log_hooks_result[build_log_hook.get_name()]`?